plugin-modernizer-core/src/main/resources/recipe_data.json

```diff
@@ -0,0 +1,12 @@
+{
+  "recipes": {
+    "PomParser": {
+      "fqcn": "io.jenkins.tools.pluginmodernizer.core.extractor.PomParser",
+      "artifactCoordinates": "io.jenkins.plugin-modernizer:plugin-modernizer-core:999999-SNAPSHOT"
+    },
+    "AddPluginsBom": {
+      "fqcn": "org.openrewrite.jenkins.AddPluginsBom",
+      "artifactCoordinates": "org.openrewrite.recipe:rewrite-jenkins:0.8.1"
```

Comment:
Something that I was wondering if it make sense to create an uber jar that contains all their recipes and their dependencies and always consume them from plugin-modernizer-core. To be discussed later

Reply:
seems like a good idea